<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="seqLayout">
      <item>
       <widget class="QComboBox" name="projectBox">
        <item>
         <property name="text">
          <string>--Select Project--</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="epBox">
        <item>
         <property name="text">
          <string>--Select Episode--</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="seqBox">
        <item>
         <property name="text">
          <string>--Select Sequence--</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="shotPlannerButton">
      <property name="toolTip">
       <string>Show Shot Planner</string>
      </property>
      <property name="text">
       <string>Shot Planner</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add selected objects to the Shot current camera corresponds</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remve selected objects from the current Shot</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="btnsLayout">
      <item>
       <widget class="QPushButton" name="addAssetsButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add Assets for the specified Sequence</string>
        </property>
        <property name="text">
         <string>Reference Assets</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Create cameras for selected shots</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
